src/posthog-surveys-types.ts

```diff
@@ -123,7 +123,7 @@ export interface SurveyResponse {
 
 export type SurveyCallback = (surveys: Survey[]) => void
 
-export type SurveyUrlMatchType = 'regex' | 'not_regex' | 'exact' | 'is_not' | 'icontains' | 'not_icontains'
+export type SurveyMatchType = 'regex' | 'not_regex' | 'exact' | 'is_not' | 'icontains' | 'not_icontains'
```

Comment:
i think we could rename this to something generic and kill `WebExperimentUrlMatchType` as well

Reply:
but then this needs to be moved out of the surveys bundle

Reply:
I think this makes sense. However, since it's generic, I'd say we name it `UrlMatchType` and move it to `src/types.ts`

Reply:
No `Url` in its name, since its not used only for Urls

Reply:
oooh, we were working towards something similar in replay 👀 🤔

Reply:
URL matching and the like